_health-care/_js/components/financial-assessment/ChildInformationSection.jsx

```diff
@@ -66,9 +71,10 @@ class ChildInformationSection extends React.Component {
         {notRequiredMessage}
 
         <div className="input-section">
-          <ErrorableCheckbox
+          <ErrorableRadioButtonGroup
```

Comment:
Right, I think it wasn't required before when it was a checkbox because not checking a checkbox implies "No", but with a radio button you have to select something to answer the question.